master/internal/command/command.go

```diff
@@ -158,7 +196,12 @@ func (c *command) Receive(ctx *actor.Context) error {
 			Notebook: c.toNotebook(ctx),
 			Config:   protoutils.ToStruct(c.Config),
 		})
-
+	case *apiv1.IdleNotebookRequest:
```

Comment:
my only real critique is that I may call this "IdleTaskRequest" or "IdleAllocationRequest" instead.

Reply:
This is only under `/api/v1/notebooks/` now and internal. I think it's fine to do so for now.